### PR TITLE
the modulations buttons should be narrower to the value display

### PR DESCRIPTION
**Pull request by hhoegelo**

### CODE REVIEW

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/world/ChildrenOwner.java [hunk 1]
```diff
@@ -49,10 +49,12 @@ public void removeAll() {
 
 	public void draw(Context2d ctx, int invalidationMask) {
 		for (TChild c : getChildren()) {
-			if (c.isVisible())
+			if (c.isVisible()) {
 				c.draw(ctx, invalidationMask | c.resetPendingInvalidations());
-			else
+				// c.getPixRect().stroke(ctx, 1, RGB.red()); // debug feature
```
Comment (schlenkibus) [inline NonMaps/src/com/nonlinearlabs/NonMaps/client/world/ChildrenOwner.java]: //debug feature
please remove commented-out code.

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/belt/parameters/BeltParameterLayout.java [hunk 2]
```diff
@@ -137,7 +136,7 @@ public void doLayout(double x, double y, double w, double h) {
 		layouter.push(mcAmountRadioButton, modulationButtonWidth, modulationButtonWidth, 1, 2);
 		layouter.push(null, margin, margin, 0, 2);
 
-		layouter.push(valueDisplay, modulationButtonWidth, sliderWidth, 2, 1);
+		layouter.push(valueDisplay, modulationButtonWidth, 3 * sliderWidth / 4, 2, 1);
```
Comment (schlenkibus) [inline NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/belt/parameters/BeltParameterLayout.java]: why not sliderWidth * 0.75 ?

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/world/ChildrenOwner.java (hunk 1 above)
Comment (hhoegelo) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/world/ChildrenOwner.java]: It is meant to stay in the code to easily enable it (by commenting in) if finding a bug requires a border around each control.

File: NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/belt/parameters/BeltParameterLayout.java (hunk 2 above)
Comment (hhoegelo) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/belt/parameters/BeltParameterLayout.java]: hmm for me its easier to read like this but I don't care much - would it make you happier if I changed it?

Comment (schlenkibus) [reply NonMaps/src/com/nonlinearlabs/NonMaps/client/world/overlay/belt/parameters/BeltParameterLayout.java]: you would make me the happiest schlenkibus ever

**Pull request merged**